tests/p2p/bootstrap_test/config.yaml

```diff
@@ -10,8 +10,8 @@ deployment_ready_time_out: 120 #default 120
 config_path: '../config.toml'
 
 bootstrap:
-  image: 'spacemeshos/go-spacemesh:develop'
-  command: '/bin/go-p2p'
+  image: 'spacemeshos/go-spacemesh-dev:harness'
```

Comment:
dont push these changes. image should stay develop

Reply:
fixed in 704b0c0b747b960ba1d5920e85d81182eaa660a8